src/Tests/Backend/Backend.test.ts

```diff
@@ -48,44 +48,51 @@ class BackendMockup implements Backend {
 }
 
 suite("Backend", function () {
+  setup(function () {
+    Object.keys(globalBackendMap).forEach(
+      (key) => delete globalBackendMap[key]
+    );
+    Object.keys(gToolchainEnvMap).forEach(
+      (key) => delete gToolchainEnvMap[key]
+    );
+  });
+
+  teardown(function () {
+    Object.keys(globalBackendMap).forEach(
+      (key) => delete globalBackendMap[key]
```

Comment:
IMO It's better if we provide delete function for backend, which recursively deleting toolchain and executors. 
Maybe you can add TODO.
I see that you have added the test hook to clean global variables for each test, to prevent the tests from affecting each others.

Reply:
The backend is basically installed by sub extension, so if you want to delete the backend, just remove the sub extension. However, it seems that addtional discussion is needed about removing ONE and EdgeTPU backend, which are installed by ONE-vscode. We will consider providing the delete function later.

Reply:
For now, ONE-vscode always has two toolchain installed, ONE and EdgeTPU. Therefore, it seems correct to implement the Backend test with the two installed.

Reply:
I understand :-D
I will close this PR and create a new PR that implements the Backend test code with the ONE and EdgeTPU installed.